cas-server-webapp/src/main/resources/templates/fragments/loginsidebar.html

```diff
@@ -22,8 +22,10 @@ <h3 th:utext="#{screen.welcome.label.loginwith}" />
                         <li th:case="'cas'" th:class="${'fa fa-sign-in'}">
                             <a th:href="${entry.redirectUrl}" th:text="${entry.name}" />
                         </li>
+                        <li th:case="*" th:class="${'fa fa-sign-in'}">
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5005199/15034398/e2390f1a-12a7-11e6-81d0-791dc00f174b.png)


Yes, I configured a client through the spring, of course I want to be displayed on the screen.
Even a little ugly does not matter, at least I know my configuration is correct.